docs/kyma/04-02-local-installation.md

```diff
@@ -23,84 +24,99 @@ Virtualization:
-
-7. Start the installation. Trigger the `run.sh` script to start Minikube with a Kyma-specific configuration and install the necessary components. Define the password used to log in to the Console UI using the `--password` flag. Run:
-    <div tabs>
-      <details>
-      <summary>
-      MacOS
-      </summary>
-
-      ```
-      ./cmd/run.sh --password {USER_PASSWORD}
-      ```
-      </details>
-      <details>
-      <summary>
-      Linux
-      </summary>
-
-      ```
-      ./cmd/run.sh --password {USER_PASSWORD} --vm-driver virtualbox
-      ```
-      </details>
-    </div>
-
-8. By default, the Kyma installation is a background process, which allows you to perform other tasks in the terminal window. Nevertheless, you can track the progress of the installation by running the `is-installed.sh` script. It is designed to give you clear information about the Kyma installation. Run it at any point to get the current installation status, or to find out whether the installation is successful.
-
-  ```
-  ./scripts/is-installed.sh
-  ```
-  >**TIP:** If the script indicates that the installation failed, try to install Kyma again by re-running the `run.sh` script. If the installation fails in a reproducible manner, don't hesitate to create a [GitHub](https://github.com/kyma-project/kyma/issues) issue in the project or reach out to the ["installation" Slack channel](https://kyma-community.slack.com/messages/CD2HJ0E78) to get direct support from the community.
-
-9. After the installation is completed, you can access the Console UI. Go to [this](https://console.kyma.local) address and select **Login with Email**. Use the **admin@kyma.cx** email address and the password you set using the `--password` flag.
-
-10. At this point, Kyma is ready for you to explore. See what you can achieve using the Console UI or check out one of the [available examples](https://github.com/kyma-project/examples).
+Follow there instructions to install Kyma from a release or from sources:
+<div tabs>
+  <details>
+  <summary>
+  From a release
+  </summary>
+    
+  1. Provision a kubernetes cluster on minikube ready to be installed with Kyma run this command:
+    
+     ```bash
+     kyma provision minikube
+     ```
+     > **NOTE:** Kyma CLI command above uses default minikube vm driver installed for your OS, for list of default vm driver for supported OS and other available vm drivers you can read from [here](http://github.com/kyma-project/cli)
+  2. Install Kyma with latest release, run following command:
+     ```bash
+     kyma install
+     ```
+     > **NOTE** If you need install a specific Kyma release use following steps
+     
+  3. Go to [this](https://github.com/kyma-project/kyma/releases/) page and choose the latest release.  
```

Comment:
Now steps 3.-5. are optional and related to the notes. Maybe @bszwarc can help here to make it more nice?